detection_rules/custom_rules.py

```diff
@@ -43,23 +43,34 @@ def create_config_content() -> str:
     return yaml.safe_dump(config_content, default_flow_style=False)
 
 
-def create_test_config_content() -> str:
+def create_test_config_content(enable_prebuilt_tests: bool) -> str:
     """Generate the content for the test_config.yaml with special content and references."""
+    comment_char = '#' if enable_prebuilt_tests else ''
     example_test_config_path = DEFAULT_CONFIG_PATH.parent.joinpath("example_test_config.yaml")
-    content = f"# For more details, refer to the example configuration:\n# {example_test_config_path}\n" \
-              "# Define tests to explicitly bypass, with all others being run.\n" \
-              "# To run all tests, set bypass to empty or leave this file commented out.\n\n" \
-              "unit_tests:\n  bypass:\n#  - tests.test_all_rules.TestValidRules.test_schema_and_dupes\n" \
-              "#  - tests.test_packages.TestRegistryPackage.test_registry_package_config\n"
+    
+    lines = [
+        "# For more details, refer to the example configuration:",
+        f"# {example_test_config_path}",
+        "# Define tests to explicitly bypass, with all others being run.",
+        "# To run all tests, set bypass to empty or leave this file commented out.",
+        "",
```

Comment:
Should this just be a \n ?

Reply:
Because of the `return '\n'.join(lines)`, the "" is the equivalent to a a \n. This was just to clean up the text to potentially make it easier to add or subtract lines later.